src/main/java/in/twizmwaz/cardinal/module/modules/titleRespawn/TitleRespawnBuilder.java

```diff
@@ -0,0 +1,32 @@
+package in.twizmwaz.cardinal.module.modules.titleRespawn;
+
+import in.twizmwaz.cardinal.match.Match;
+import in.twizmwaz.cardinal.module.BuilderData;
+import in.twizmwaz.cardinal.module.ModuleBuilder;
+import in.twizmwaz.cardinal.module.ModuleCollection;
+import in.twizmwaz.cardinal.module.ModuleLoadTime;
+import in.twizmwaz.cardinal.util.StringUtils;
+import org.jdom2.Element;
+
+@BuilderData(load = ModuleLoadTime.NORMAL)
+public class TitleRespawnBuilder implements ModuleBuilder {
+
+    @Override
+    public ModuleCollection load(Match match) {
+        Element head = match.getDocument().getRootElement().getChild("respawn");
```

Comment:
But this module is enabled on every Overcast map; it's a default OCN respawn system now.